database/docker-compose.yml

```diff
@@ -0,0 +1,46 @@
+version: '3.8'
+
+services:
+
+  postgres:
```

Comment:
can you please give the actual name?
like accountdb similarly for the below one as well?

Reply:
Sure, I have updated the names to Accountsdb & Transactionsdb.